src/server/google-cloud.md

```diff
@@ -6,8 +6,8 @@ description: Your Dart app can use many Google Cloud services — Firebase, Goog
 
 Dart servers can use many
 [Google Cloud products](https://cloud.google.com/products),
-often with the help of pre-packaged [Docker images][]
-that the Dart team maintains.
+often with the help of Dart's official [Docker images][]
```

Comment:
```suggestion
often with the help of [Dart Docker images][]
```

Reply:
I think "official" isn't needed, since we imply that with the "team" below.

Reply:
They are not "Dart official" but "Docker official"! 

Maybe we should say "often with the Docker Official Images for Dart"? And maybe link to https://docs.docker.com/docker-hub/official_images/ ?

Reply:
Oh, interesting! That's completely different from what I thought it meant.

Will many people care that they're Official Images? If not, I still like my wording. But someone who knows more about Docker might have a different take.

Reply:
Talking with @johnpryan, we had a couple of different possibilities:

1. ...with the help of [Dart's Docker images][], which are [Official Images][]
2. ...with the help of the official [Dart container images][], maintained by Docker, Inc.

Reply:
Can we just say this?

```
Dart servers can use many
[Google Cloud products](https://cloud.google.com/products),
often with the help of the
pre-packaged Docker [Official Images for Dart](https://hub.docker.com/_/dart).
```

Reply:
lgtm